docker-compose/config/pm2-dev.json

```diff
@@ -3,8 +3,8 @@
     "name"         : "KuzzleServer",
     "script"       : "bin/kuzzle",
     "args"         : ["start"],
-    "watch"        : ["lib", "config", "bin", "default.config.js"],
-    "ignore_watch" : ["node_modules"],
-    "node_args": "--inspect"
+    "watch"        : ["lib", "config", "bin", "default.config.js", "plugins/enabled"],
```

Comment:
nitpicking: If we are going to improve this section:

* is it worth watching the bin directory?
* shouldn't we, rather than having an ignore_watch, watch js files only (`["lib/**/*.js", "default.config.js", "plugins/enabled/**.js"]`)?